### ISSUE TITLE
Codecov publishing is broken

### ISSUE THREAD

**Issue by lenkan** [WebOfTrust/signify-ts#246]
See latest workflow runs:

```
[2024-04-11T09:31:39.774Z] ['error'] There was an error running the uploader: Error uploading to [https://codecov.io:](https://codecov.io/) Error: There was an error fetching the storage URL during POST: 404 - {'detail': ErrorDetail(string='Unable to locate build via Github Actions API. Please upload with the Codecov repository upload token to resolve issue.', code='not_found')}
[2024-04-11T09:31:39.775Z] ['info'] Codecov will exit with status code 0. If you are expecting a non-zero exit code, please pass in the `-Z` flag
```

**Comment by m00sey** [comment]
![image](https://github.com/WebOfTrust/signify-ts/assets/180035/ad721efc-9179-45ce-8a1d-511706a9aefe)

🤷

**Comment by lenkan** [comment]
Looks like your fix in #247  was sufficient to get it published again 👍
